src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -325,9 +326,7 @@ export default defineComponent({
             { bytecode },
           ] = await Promise.all([
             fetchJson(`${activeNetwork.value.url}/v3/contracts/${popupProps.value.tx.contractId}/code`),
-            // SDK is needed to establish the `networkId` and the dex contracts for the network
-            // TODO replace with `getSdk` after migration to SDK13
-            store.dispatch('sdkPlugin/initialize'),
+            getSdk(),
```

Comment:
I suggest to leave the previous comment because without it using the `getSdk` has no context.